src/modules/TftpServer.rb

```diff
@@ -11,10 +11,19 @@
 # Input and output routines.
 require "yast"
 
+require "shellwords"
+
+require "yast2/target_file" # allow CFA to work on change scr
+require "cfa/tftp_sysconfig"
+require "y2firewall/firewalld"
+
+
 module Yast
   class TftpServerClass < Module
 
-    SERVICE_NAME = "xinetd"
+    SOCKET_NAME = "tftp"
+    PACKAGE_NAME = "tftp"
+    SERVER_BIN = "in.tftp"
```

Comment:
Just curious: this seems nowhere to be used

Reply:
it was in killall and then I change it, thanks for notice